.github/workflows/rubocop.yml

```diff
@@ -0,0 +1,20 @@
+name: Lint
+
+on:
+  pull_request:
+    branches:
+      - '*'
+
```

Comment:
If we only do pull_request I believe the action will only run on the first commit to the branch and on merges. Adding `push` will also run in the cop on subsequent code sommits